src/lib/blog/BlogPostFooter.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+	import type {FeedItemData} from '$lib/blog/blogFeed';
+
+	export let post: FeedItemData;
+</script>
+
+<p>Want to discuss this post privately?</p>
```

Comment:
This phrasing feels weird. Maybe something more like?
```suggestion
<p>Want to let us know what you thought?</p>
```